widget/list.go

```diff
@@ -654,8 +670,16 @@ func (l *listLayout) updateList(refresh bool) {
 	l.list.scroller.Content.(*fyne.Container).Objects = objects
 	l.renderLock.Unlock() // user code should not be locked
 
-	for row, obj := range visible {
-		l.setupListItem(obj, row, focused == obj)
+	if newOnly {
+		for nId, nItem := range visible {
```

Comment:
Can you please keep the variables consistent with the loop below? I think I personally prefer row and obj for naming.